client/api_helm.go

```diff
@@ -0,0 +1,757 @@
+/*
+ * Pipeline API
+ *
+ * Pipeline v0.3.0 swagger
+ *
+ * API version: 0.3.0
+ * Contact: info@banzaicloud.com
+ * Generated by: OpenAPI Generator (https://openapi-generator.tech)
+ */
+
+package client
+
+import (
+	"context"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+	"fmt"
+	"github.com/antihax/optional"
```

Comment:
Please add this package to dependencies and vendor

Reply:
I have fixed this as well now, and have added the check for other vendoring issues! Thanks for catching this.